src/index.ts

```diff
@@ -153,6 +179,19 @@ const getElementPosition = (el: Element): ElementPositionWithDomNode => {
   };
 };
 const setElementPosition = (el: Element, value: ElementPositionWithDomNode) => {
+  if (
+    value.insertBeforeNode &&
+    !value.parentNode.contains(value.insertBeforeNode)
+  ) {
+    console.error(
+      'position mutation skipped - insertBeforeNode not a child of parent',
+      {
+        el,
+        ...value,
+      }
+    );
```

Comment:
In what scenario would this happen?

Reply:
The only scenario that I've observed is when an individual element's mutation observer fires and tries to put itself back into a position that is out of sequential order (relative to other position mutations) and thereby doesn't exist - in those cases, we would not want to run the mutation anyway.